### PR TITLE
[WIP] Add support for Crashpad/Breakpad minidump

### PR DESCRIPTION
**Pull request by lucasg**
This is a work-in-progress, so feel free to merge it into the main tree at your leisure.

Breakpad and Crashpad generate minidumps on crash, but also add "user-specific" streams since, well, they have user-specific data. I didn't attempt to parse those streams yet, but I had to relax the stream directory parsing in order to skip unknown streams.

### CODE REVIEW

File: minidump/minidumpfile.py [hunk 1]
```diff
@@ -183,10 +192,12 @@ def __parse_header(self):
 		self.header = MinidumpHeader.parse(self.file_handle)
 		for i in range(0, self.header.NumberOfStreams):
 			self.file_handle.seek(self.header.StreamDirectoryRva + i * 12, 0 )
-			self.directories.append(MINIDUMP_DIRECTORY.parse(self.file_handle))
-
+			minidump_dir = MINIDUMP_DIRECTORY.parse(self.file_handle)
+			if minidump_dir:
+				self.directories.append(minidump_dir)
```
Comment (skelsec) [inline minidump/minidumpfile.py]: Please add a log line here that there is a stream that was ignored of type X

Comment (skelsec) [reply minidump/minidumpfile.py]: but actually it's not needed

Comment (lucasg) [reply minidump/minidumpfile.py]: ok done.

**Comment by skelsec** [comment]
Thank you for the contribution. In case you progress with the parsing pls submit a PR! :)

**Pull request merged**